Library/Homebrew/download_strategy.rb

```diff
@@ -262,6 +262,10 @@ def initialize(name, resource)
   end
 
   def fetch
+    if @url.include?("https://downloads.sourceforge.net") || @url.include?("https://download.gnome") || @url.include?("https://www.apache.org/dyn/")
```

Comment:
Wouldn't `https://download.gnome.org` be better than `https://download.gnome` here? `.com` and `.net` aren't used by the GNOME project, though I haven't checked other TLDs.